framework/core/views/install/update.php

```diff
@@ -18,37 +18,36 @@
 </form>
 
 <script>
-  document.querySelector('form input').select();
-
-  document.querySelector('form').addEventListener('submit', function(e) {
-    e.preventDefault();
-
-    var button = this.querySelector('button');
-    button.textContent = 'Please Wait...';
-    button.disabled = true;
-
-    fetch('', {
-      method: 'POST',
-      body: new FormData(this)
-    })
-      .then(response => {
-        if (response.ok) {
-          window.location.reload();
-        } else {
-          response.text().then(errorMessage => {
-            var error = document.querySelector('#error');
-            error.style.display = 'block';
-            error.textContent = 'Something went wrong:\n\n' + errorMessage;
-            button.disabled = false;
-            button.textContent = 'Update Flarum';
-          });
-        }
-      })
-      .catch(error => {
-        console.error('Error:', error);
-      });
+  window.onload = function() {
```

Comment:
https://stackoverflow.com/a/37401912 might be a better fit, actually

Reply:
let me just switch back to jQuery 🤣

Reply:
Aaaaaaaaa noooooo!!!!!!!